### PR TITLE
kajas blogpost from team berlin diamonds. images were already compres…

### PR DESCRIPTION
**Pull request by kajatiger**
…sed, I couldn't make them any smaller.

### CODE REVIEW

**Comment by carpodaster** [comment]
@kajatiger the images are 5k x 3.5k pixels. Can you resize them to a width of about 700px? That should shed a few kilobytes.

**Comment by kajatiger** [comment]
when is the blog post going to be deployed and online?

**Comment by alicetragedy** [comment]
Hi @kajatiger, I will review this tomorrow morning, so if it's ready to go it will be published and shared tomorrow. :)

File: blog/_posts/2017-07-25-team-berlindiamonds-part-one.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Kaja
+
+I am a passionate Ruby developer and a member of the Rails Girls Berlin. Originally I studied philosophy and historical linguistics, but I soon discovered my love for programming and am now a junior Rails developer. Besides coding I also love playing football, eating and learning new stuff. Oh and I am addicted to the internet in general. I just learned to accept it and use that in a good way.
+
+
+Jen
+
+Since I originally come from photography and psychology, I love to connect creativity and coding. So building web apps with Rails is just the right medium for me. I also joined the Opend Tech School Berlin and soon met the Rails Girls Berlin at some events at our co-working space. I love to travel and see beautiful things and I can do some tricks on the skateboard.
```
Comment (alicetragedy) [inline blog/_posts/2017-07-25-team-berlindiamonds-part-one.md]: minor typo: Opend -> Open

File: blog/_posts/2017-07-25-team-berlindiamonds-part-one.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: Creating our own Plugin, Yeeey!
+layout: post
+created_at: Tue Jul 25 2017
+permalink: blog/2017-07-25-team-berlindiamonds
+current: blog
+author: Kaja
+twitter: AlizeNero
+categories:
+- news
```
Comment (alicetragedy) [inline blog/_posts/2017-07-25-team-berlindiamonds-part-one.md]: change category to `student-posts`

File: blog/_posts/2017-07-25-team-berlindiamonds-part-one.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+![image](../img/blog/2017/2017-07-firstday-confetti.jpg)
```
Comment (alicetragedy) [inline blog/_posts/2017-07-25-team-berlindiamonds-part-one.md]: Could you please add an image credit for this? (i.e. who took the picture, like you did for the second picture)

File: blog/_posts/2017-07-25-team-berlindiamonds-part-one.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+Our Plugin
+
+We already got quite far within the first weeks and finished building a plugin for Discourse that can upload backup files to Google Drive. If you want to check out the code and see our progress, please take a look at our [Github Repo](https://github.com/berlindiamonds/discourse-googledrive-backup). Although the plugin itself is technically working, as good coders we will still refactor our code and also create a reusable structure in order to build more backup plugins after that. 
+All in all this project has really made us feel more competent and growing with every problem we solve. We feel very empowered and happy and are very thankful for the chance of being part of the RailsGirls Summer of Code. Thanks to all contributers and organizers! We love it <3 
+![image](../img/blog/2017/2017-07-empowered-diamonds.jpg) Photo by: Markus from Absolventa
```
Comment (alicetragedy) [inline blog/_posts/2017-07-25-team-berlindiamonds-part-one.md]: we've got a special `<div class="image-credits">` for all image credits, could you add this to the post for the two images? you can see what it looks like in other teams' blog posts :) thanks!

**Review by alicetragedy — CHANGES_REQUESTED** [review]
hey @kajatiger, thanks for your contribution! There are a couple of minor things that need changing (adding image credits, adding styling for the credits) before we merge and share the post. Would you have time to do this today still? Thanks! 🎉

File: blog/_posts/2017-07-25-team-berlindiamonds-part-one.md (hunk 4 above)
Comment (kajatiger) [reply blog/_posts/2017-07-25-team-berlindiamonds-part-one.md]: ah I was already wondering if there was some convention like that, thanks! I think I made all the changes now.

File: blog/_posts/2017-07-25-team-berlindiamonds-part-one.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+permalink: blog/2017-07-25-team-berlindiamonds
+current: blog
+author: Kaja
+twitter: AlizeNero
+categories:
+- student-posts
+---
+
+![image](../img/blog/2017/2017-07-firstday-confetti.jpg)
+<div class="image-credits">Kaja at her desk / Photo by: Jen</div>
```
Comment (alicetragedy) [inline blog/_posts/2017-07-25-team-berlindiamonds-part-one.md:14]: 👍

File: blog/_posts/2017-07-25-team-berlindiamonds-part-one.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+Our Plugin
+
+We already got quite far within the first weeks and finished building a plugin for Discourse that can upload backup files to Google Drive. If you want to check out the code and see our progress, please take a look at our [Github Repo](https://github.com/berlindiamonds/discourse-googledrive-backup). Although the plugin itself is technically working, as good coders we will still refactor our code and also create a reusable structure in order to build more backup plugins after that. 
+All in all this project has really made us feel more competent and growing with every problem we solve. We feel very empowered and happy and are very thankful for the chance of being part of the RailsGirls Summer of Code. Thanks to all contributers and organizers! We love it <3 
+![image](../img/blog/2017/2017-07-empowered-diamonds.jpg)
+<div class="image-credits">Kaja and Jen / Photo by: Markus from Absolventa</div>
```
Comment (alicetragedy) [inline blog/_posts/2017-07-25-team-berlindiamonds-part-one.md:44]: 👍

**Review by alicetragedy — APPROVED** [review]
perfect, thanks @kajatiger! this is ready to go 🎉

**Pull request merged**